crates/runc-shim/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "containerd-shim-runc-v2"
+version = "0.1.0"
+authors = ["Shaobao Feng <fshb1988@gmail.com>", "Tianyang Zhang <burning9699@gmail.com>"]
+edition = "2018"
+license = "Apache-2.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+async = []
```

Comment:
Async is not currently implemented in this PR, right? Should remove this flag?

Reply:
Just for reserved, remove it also OK

Reply:
distinguish from "sync" feature. There would be two same funcs if removed.